moto/sqs/responses.py

```diff
@@ -227,7 +227,9 @@ def send_message(self):
         if len(message) > MAXIMUM_MESSAGE_LENGTH:
             return ERROR_TOO_LONG_RESPONSE, dict(status=400)
 
-        message_attributes = parse_message_attributes(self.querystring)
+        message_attributes, aws_trace_header = parse_message_attributes(
```

Comment:
Looks like the `parse_message_attributes` now has completely duplicate logic. Would be better if we can reuse the same method:
```
message_attributes = parse_message_attributes(self.querystring)  # default value for key will be "MessageAttribute"
system_attributes = parse_message_attributes(self.querystring, key="MessageSystemAttribute")
```

Reply:
Done,Changed as requested